e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -205,7 +205,7 @@ databricks_host=https://$(echo "$arm_output" | jq -r '.properties.outputs.databr
 databricks_workspace_resource_id=$(echo "$arm_output" | jq -r '.properties.outputs.databricks_id.value')
 databricks_aad_token=$(az account get-access-token --resource 2ff814a6-3304-4ab8-85cb-cd0e6f879c1d --output json | jq -r .accessToken) # Databricks app global id
```

Comment:
In a future PR, please adjust even things like this.  `databricks_aad_token` -> `databricks_access_token`

I won't block the PR from merging over things like this, but would be good to finish getting cleaned up.